pkg/kubelet/config/file_linux_test.go

```diff
@@ -128,6 +128,15 @@ func TestWatchFileChanged(t *testing.T) {
 	}
 }
 
+func TestWatchFileChangedByFsNotify(t *testing.T) {
+	for _, testCase := range testCases {
+		if testCase.watchDir && testCase.symlink {
+			continue
```

Comment:
Exclude this case because of https://github.com/kubernetes/kubernetes/pull/99926#discussion_r589551746